src/Workspaces/Core/Portable/Rename/ConflictEngine/RenamedSpansTracker.cs

```diff
@@ -171,11 +178,11 @@ internal async Task<Solution> SimplifyAsync(Solution solution, IEnumerable<Docum
                         complexifiedSpans.Clear();
                     }
 
-                    var root = await document.GetSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
+                    var root = await document.GetRequiredSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
 
                     // First, get all the complexified statements
                     var nodeAnnotations = renameAnnotations.GetAnnotatedNodesAndTokens<RenameNodeSimplificationAnnotation>(root)
-                        .Select(x => Tuple.Create(renameAnnotations.GetAnnotations<RenameNodeSimplificationAnnotation>(x).First(), (SyntaxNode)x));
+                        .Select(x => Tuple.Create(renameAnnotations.GetAnnotations<RenameNodeSimplificationAnnotation>(x).First(), (SyntaxNode)x!));
```

Comment:
Here it is safe because `RenameNodeSimplificationAnnotation` is only used to annotated SyntaxNode